### PR TITLE
Enable history tracking on create and destroy by default

### PR DESCRIPTION
**Pull request by JagdeepSingh**
Bump next version to 0.8.0

This is a replacement for https://github.com/mongoid/mongoid-history/pull/207.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
-### 0.7.1 (Next)
+### 0.8.0 (Next)
 
 * [#208](https://github.com/mongoid/mongoid-history/pull/208): Fix: history tracks fields declared after `track_history` - [@mikwat](https://github.com/mikwat).
+* [#211](https://github.com/mongoid/mongoid-history/pull/211): BREAKING CHANGE - Enable tracking create/destroy by default - [@jagdeepsingh](https://github.com/jagdeepsingh).
```
Comment (dblock) [inline CHANGELOG.md]: Lets not be dramatic :) It's not a breaking change, it's just a different behavior. I would just remove the BREAKING CHANGE part and call it a day.

File: spec/unit/trackable_spec.rb [hunk 2]
```diff
@@ -613,6 +615,7 @@ def self.name
     let(:m) { MyModel.create!(foo: 'bar') }
 
     it 'should create history' do
+      m   # Created
```
Comment (dblock) [inline spec/unit/trackable_spec.rb]: Instead use `let!`, it will create `m`.

**Comment by johnnyshields** [comment]
@JagdeepSingh please handle

**Comment by JagdeepSingh** [comment]
@dblock done.

**Comment by johnnyshields** [comment]
@dblock please merge

**Comment by dblock** [comment]
👍

**Pull request merged**